runtime/src/main/java/io/quarkus/jsch/runtime/JSchSessionRuntimeConfig.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.jsch.runtime;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.WithDefault;
+
+@ConfigGroup
+public interface JSchSessionRuntimeConfig {
+
+    /**
+     * The host to use for the JSch session.
+     */
+    @WithDefault("localhost")
```

Comment:
I don't think we should provide a default here, as the host should be required and assuming `localhost` doesn't make much sense given that you won't be normally running a SSH server locally

Reply:
fixed